horace_core/sqw/@sqw/private/cut_accumulate_data_.m

```diff
@@ -168,3 +199,43 @@
     s(no_pix) = 0;
     e(no_pix) = 0;
 end
+
+
+function pci = init_pix_combine_info(nfiles, nbins)
+    % Create a pix_combine_info object to manage temporary files of pixels
+    wk_dir = get(parallel_config, 'working_directory');
+    gen_fpath = @(x) fullfile( ...
+        wk_dir, ['horace_subcut_', rand_digit_string(16), '.tmp'] ...
```

Comment:
This does not look as if it guarantees all strings to be different. Highly likely that they are but not certain.

Reply:
If absolute uniqueness is required `java.util.UUID.randomUUID()`; would provide that at the cost of a longer string.